kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-integ-tests-sample/pom.xml

```diff
@@ -15,6 +15,27 @@
     <start-class>org.kie.server.springboot.samples.KieServerIntegrationTestsApplication</start-class>
   </properties>
 
+  <dependencyManagement>
+    <dependencies>      
+      <dependency>
+        <groupId>org.webjars</groupId>
+        <artifactId>swagger-ui</artifactId>
+        <version>2.2.10</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.cxf</groupId>
+        <artifactId>cxf-rt-rs-service-description-swagger</artifactId>
+        <version>${version.org.apache.cxf}</version>
+      </dependency>
+      <!-- Notice that higher versions like 1.5.20 contain a bug (No message body writer has been found) so downgraded to 1.5.15 -->
```

Comment:
Can you please move the version declaration to one location which can be inherited here and in kie-server-spring-boot-sample? So there will be a single source of the version declaration.

Reply:
Done, moved to _kie-spring-boot-samples_ and deleted from the other 2 children.